Library/Formula/gnuradio.rb

```diff
@@ -138,7 +147,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
 
     res = %w[Markdown Cheetah lxml numpy]
-    res << %w[sphinx sphinx_rtd_theme alabaster babel docutils pygments
+    res += %w[sphinx sphinx_rtd_theme alabaster babel docutils pygments
```

Comment:
Please leave `<<` here, it modifies `res` in-place while `+=` creates a new array then assign it to `res`.